README.md

```diff
@@ -34,7 +34,312 @@ This README documents the build, customisation and testing of these runtime imag
 
 **Do you want to learn more about using Node.js actions to build serverless applications?** Please see the main project documentation [here](https://github.com/apache/openwhisk/blob/master/docs/actions-nodejs.md) for that information.
 
-## Usage
+## Build Runtimes
+
+### You have 2 options to build the NodeJS runtime:
+- Building locally
```

Comment:
Using Gradle should be the featured option; the manual Docker build which is added in this PR should come after.  Most of the OpenWhisk project use Gradle Wrapper (gradlew) because the build complexity does not need to distract from the developer fixing a bug in the runtime code itself.